build.gradle

```diff
@@ -16,71 +11,168 @@ buildscript {
         }
     }
     dependencies {
-        classpath 'net.minecraftforge.gradle:ForgeGradle:1.2-SNAPSHOT'
+        classpath 'net.minecraftforge.gradle:ForgeGradle:2.0-SNAPSHOT'
     }
 }
 
-configurations {
-    deployJars
-}
+apply plugin: "maven"
+apply plugin: "net.minecraftforge.gradle.patcher"
 
-import static net.minecraftforge.gradle.dev.ForgeDevPlugin.*
-
-apply plugin: 'maven'
-apply plugin: 'forgedev'
+minecraft {
+    version = "1.8"
+    //mappings = 'stable_16'
```

Comment:
What intentional?
 - removal of the old 'maven' and 'forgedev' plugin definitions?  `yes`
 - setting the MC version to 1.8? `yes`
- the commented out 'stable_16' `no, but that doesnt really matter does it?`